lib/compiler/src/beam_ssa.erl

```diff
@@ -404,26 +404,24 @@ common_dominators(Ls, Dom, Numbering) ->
     Doms = [map_get(L, Dom) || L <- Ls],
     dom_intersection(Doms, Numbering).
 
--spec fold_instrs_rpo(Fun, From, Acc0, Blocks) -> any() when
+-spec fold_instrs_rpo(Fun, Labels, Acc0, Blocks) -> any() when
```

Comment:
This name no longer makes sense. The blocks will be visited in the order of the labels in `Labels`. I suggest you rename it to `fold_instrs`. Also, while at it, please add comment in the style of the other comments in this module.